CS/CASESTUDY1/Bike.java

```diff
@@ -0,0 +1,41 @@
+public class Bike extends Vehicle {
+
+	private boolean selfStart;     		// bike is self start or not
+
+	private double helmetPrize;			// helmet will be gifted with bike or not
+
+	public boolean isSelfStart() {
+		return selfStart;
+	}
+
+	public void setSelfStart(boolean selfStart) {
+		this.selfStart = selfStart;
+	}
+
+	public double getHelmetPrize() {
+		return helmetPrize;
+	}
+
+	public void setHelmetPrize(double helmetPrize) {
+		this.helmetPrize = helmetPrize;
+	}
+
+	@Override
+	public double calculateOnRoadPrice() {
+		double onRoadPrice = 0;
+		if (isSelfStart()) {						// if bike is self start than increase road tax with 1000
+			onRoadPrice = onRoadPrice + 1000;
```

Comment:
Why do you have to write onRoadPrice + 1000??? when you are sure that onRoadPrice  =0 at thispoint then you can directly assign 1000 to onRoadPrice , So it should be onRoadPrice =1000